apps/block_scout_web/lib/block_scout_web/views/tokens/helper.ex

```diff
@@ -40,7 +40,8 @@ defmodule BlockScoutWeb.Tokens.Helper do
   end
 
   # TODO: remove this clause along with token transfer denormalization
-  defp do_token_transfer_amount(%Token{type: "ERC-20", decimals: nil}, nil, amount, _amounts, _token_ids) do
+  defp do_token_transfer_amount(%Token{type: type, decimals: nil}, type, amount, _amounts, _token_ids)
```

Comment:
This clause is needed for cases when token transfers denormalization is not finished yet, so this line should look like this (second parameter is `nil`). And the next clause should match on `ERC-404` as well
```suggestion
  defp do_token_transfer_amount(%Token{type: type, decimals: nil}, nil, amount, _amounts, _token_ids)
```